src/HistoryMessage.tsx

```diff
@@ -23,8 +20,13 @@ export const HistoryMessage = (props: {
             inside = <Attachment attachment={props.message.channelData.attachments[0]} actions={props.actions} />;
         }
 
-    } else
-        inside = <span>{ textify(props.message.text) }</span>
+    } else if (props.message.text) {
+        // TODO @eanders-MS: Send format={ props.activity.textFormat } once updated to DirectLine v3
```

Comment:
if possible would prefer to pass the textFormat now, we don't need DirectLine. We can get it from channelData if it's there and default to plain otherwise. See above.